packages/app/src/cli/models/app/extensions.ts

```diff
@@ -28,8 +28,19 @@ export const UIExtensionConfigurationSchema = schema.define.object({
     )
     .default([]),
   extensionPoints: schema.define.array(schema.define.string()).optional(),
-  capabilities: schema.define.any().optional(),
-
+  capabilities: schema.define
+    .object({
+      block_progress: schema.define.boolean().default(false),
+      network_access: schema.define.boolean().default(false),
+    })
+    .default({
+      block_progress: false,
+      network_access: false,
```

Comment:
Can we actually remove these defaults? I think having these values in the config will actually break extensions validation during deploy for all except Checkout UI extensions. We haven't standardized support for these yet